examples/twittermap/web/public/javascripts/app.js

```diff
@@ -1,4 +1,5 @@
-var app = angular.module('cloudberry', ['cloudberry.map','cloudberry.timeseries','cloudberry.sidebar', 'cloudberry.util','cloudberry.cache']);
+var app = angular.module('cloudberry', ['cloudberry.map','cloudberry.timeseries','cloudberry.sidebar', 'cloudberry.util','cloudberry.cache',
+                          'cloudberry.resultcache']);
```

Comment:
Since we will possibly have another cache for timeseries result and hashtag results,  let’s change the name of cache module from resultcache to mapresultcache.

Reply:
I have made changes to the names.